openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/dynamic_constraints.hpp

```diff
@@ -52,12 +53,14 @@ struct DynamicConstraints
 
   const Double max_speed;
 
+  const Double max_jerk;
```

Comment:
maxJerk field does not exist in [DynamicConstraints.](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/DynamicConstraints.html)
Please remove it.